### PR TITLE
Add view vw_file

### PR DESCRIPTION
**Pull request by max-trolliet**
Still need the function DELETE. 

The UPDATE and INSERT function run in postgis. However, they seem to contain errors as it is not working in qgis.

### CODE REVIEW

File: view/vw_file_full.sql [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+-- DROP VIEW qgep.vw_file;
+
+CREATE OR REPLACE VIEW qgep.vw_file AS 
+ SELECT f.obj_id,
+    f.identifier,
+    f.kind AS file_kind,
+    f.object,
+    f.class,
+    dm.kind AS data_media_kind,
```
Comment (m-kuhn) [inline view/vw_file_full.sql]: @urskaufmann what aliases should be set here (for f.kind and dm.kind)?

**Comment by max-trolliet** [comment]
added delete function

**Comment by m-kuhn** [comment]
did you close this on purpose?

**Comment by max-trolliet** [comment]
No... ^^ i made a mistake here and don't know how to reopen it...

**Pull request closed without merging**